Dockerfile_multi.j2

```diff
@@ -0,0 +1,21 @@
+{%- set modname = "mediatimestamp" -%}
+{%- import 'static-commontooling/docker/Dockerfile_multi_macros.j2' as macros -%}
+
+{{ macros.preamble() }}
+
+{{ macros.layer(modname, entrypoint=False) }}
+
+{{ macros.testdeps(modname) }}
+
+{{ macros.tests(modname) }}
+
+{{ macros.unittest(modname) }}
+
+{{ macros.flake8(modname) }}
+
+{{ macros.mypy(modname) }}
+
+{{ macros.wheel(modname) }}
+
+{{ macros.docs(modname) }}
+CMD ["--html", "--force", "--skip-errors", "--output-dir", ".", "{{ modname }}"]
```

Comment:
Are these errors that could or should be fixed at some point?

Reply:
They're import errors of external libs which only appear on docs generation and don't seem to impact the generation of docs. Happy if you want to drop a job in pivotal to investigate if it can be sorted properly.